src/components/Logo.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import { Link } from 'components';
-import { RoadieRIcon } from 'components/icons';
+import { RoadieWordIcon, RoadieRIcon } from 'components/icons';
 
 const Logo = () => (
   <Link to="/">
     <span className="sr-only">Roadie</span>
-    <RoadieRIcon />
+    <span className="md:block hidden">
+      <RoadieWordIcon />
+    </span>
+    <span className="md:hidden">
+      <RoadieRIcon />
```

Comment:
But it's a bit wide on mobile screen sizes so I'm reverting to using the "R" there.